### ISSUE TITLE
Error while checking for xolvio:cucumber@0.7.1 (Windows)

### ISSUE THREAD

**Issue by ManuelDeLeon** [xolvio/meteor-cucumber#87]
```
C:\Meteor>meteor create test2
test2: created.

To run your new app:
  cd test2
  meteor
```
```
C:\Meteor>cd test2
```
```
C:\Meteor\test2>meteor add xolvio:cucumber
 => Errors while adding packages:

While checking for xolvio:cucumber@0.7.1:
error: No compatible binary build found for this package. Contact the
package author and ask them to publish it for your platform.
```

**Comment by samhatoum** [comment]
Thanks for reporting, looking into it

**Comment by ManuelDeLeon** [comment]
Au contraire mon frere, thank YOU... The latest changes look awesome.

**Comment by ManuelDeLeon** [comment]
fyi, same error for 0.7.2

**Comment by samhatoum** [comment]
I know, we're having trouble building for windows, some npm weirdness going on. trying to fix again tomorrow. Stay tuned!

**Comment by samhatoum** [comment]
Fixed in 0.8.0
